src/main/java/emu/grasscutter/server/game/GameServer.java

```diff
@@ -269,6 +270,8 @@ public void run() {
 	@Override
 	public void onStartFinish() {
 		Grasscutter.getLogger().info(translate("messages.status.free_software"));
+		Grasscutter.getLogger().info(translate("messages.status.game_version", GameConstants.VERSION));
+		Grasscutter.getLogger().info(translate("messages.status.version", BuildConfig.VERSION, BuildConfig.GIT_HASH));
```

Comment:
The BuildConfig version log message should be run when the server starts not after it finishes starting up, in case the server throws an error before startup.

Reply:
Done!